yt_dlp/extractor/dplay.py

```diff
@@ -349,7 +349,7 @@ class DiscoveryPlusIE(DPlayIE):
     _API_URL = 'us1-prod-direct.discoveryplus.com'
 
     def _update_disco_api_headers(self, headers, disco_base, display_id, realm):
-        headers['x-disco-client'] = f'WEB:UNKNOWN:{self._PRODUCT}:15.0.0'
+        headers['x-disco-client'] = f'WEB:UNKNOWN:{self._PRODUCT}:25.2.6'
```

Comment:
make sure to test all extractors that depend on `DiscoveryPlusIE`

Reply:
I did a search within the repository and saw that it was only called within dplay.py for diynetwork, sciencechannel, animalplanet and discoveryplus itself.

Tested it before commit to make sure downloads still worked for all 4 and they did, so threw the commit up here. :-)